### PR TITLE
Expend sharedInstanceWithToken

### PR DESCRIPTION
**Pull request by amitayfeder1**
Expend sharedInstanceWithToken to accept trackCrashes and automaticPushTracking

### CODE REVIEW

**Comment by amitayfeder1** [comment]
@wgins

**Comment by wgins** [comment]
@pchien when you have a chance do you mind taking a look? Thanks!

File: Mixpanel/Mixpanel.m [hunk 1]
```diff
@@ -45,8 +45,12 @@ + (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken launchOptions:(NSDict
 #else
     const NSUInteger flushInterval = 60;
 #endif
+    
+    return [[self alloc] initWithToken:apiToken launchOptions:launchOptions flushInterval:flushInterval trackCrashes:trackCrashes automaticPushTracking:automaticPushTracking];
+}
 
-    return [[self alloc] initWithToken:apiToken launchOptions:launchOptions andFlushInterval:flushInterval];
++ (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken launchOptions:(NSDictionary *)launchOptions {
+    return [Mixpanel sharedInstanceWithToken:apiToken launchOptions:launchOptions trackCrashes:YES automaticPushTracking:NO];
```
Comment (pchien) [inline Mixpanel/Mixpanel.m]: To keep behavior the same as before, let's make `automaticPushTracking:YES` default

Comment (amitayfeder1) [reply Mixpanel/Mixpanel.m]: Yes, my bad

**Pull request merged**